machine-learning/solutions/seismic_salt_detection/trainer/model.py

```diff
@@ -73,51 +73,40 @@ def create_regressor(config, parameters):
     Returns:
         A configured and ready to use `tf.estimator.DNNRegressor`
     """
-
     # Mean and Standard Deviation Constants for normalization.
-    with file_io.FileIO(parameters.mean_path, mode='r') as f:
-        mean = pickle.load(f)
-    with file_io.FileIO(parameters.std_path, mode='r') as f:
-        std = pickle.load(f)
+    mean = np.float32(parameters.depth_mean)
+    std = np.float32(parameters.depth_std)
 
     # Columns to be used as features.
-    hour = tf.feature_column.categorical_column_with_identity(
-        'hour', num_buckets=24)
-    hour = tf.feature_column.embedding_column(
-        hour, dimension=parameters.hour_embedding)
-
-    day = tf.feature_column.categorical_column_with_identity(
-        'day', num_buckets=7)
-    day = tf.feature_column.embedding_column(
-        day, dimension=parameters.day_embedding)
-
-    weather = [tf.feature_column.numeric_column(
-        'weather' + str(i),
-        normalizer_fn=(lambda x, i = i: (x - mean[i]) / std[i])
-    ) for i in range(constants.WEATHER_SIZE)]
-
-    distribution = [tf.feature_column.numeric_column(
-        'distribution' + str(i)
-    ) for i in range(constants.DISTRIBUTION_SIZE)]
-
-    feature_cols = [hour, day] + weather + distribution
+    
+    depth = tf.feature_column.numeric_column(
+        'depth',
+        normalizer_fn=(lambda x: (x - mean) / std))
+    
+    image = hub.image_embedding_column('image', parameters.module)
+    
+    feature_cols = [depth,image]
 
-    # Evaluation metric.
-    def mean_absolute_error(labels, predictions):
-        """Creates mean absolute error metric.
+    def estimator_metrics(labels, predictions):
+        """Creates metrics for Estimator.
 
-        Metric is used to evaluate the model.
+        Metrics are used to evaluate the model.
 
         Args:
             labels: Evaluation true labels.
             predictions: Evaluation model predictions.
 
         Returns:
-            A dictionary with the evaluation metric
+            A dictionary with the evaluation metrics
         """
-        pred_values = predictions['predictions']
-        return {'mae': tf.metrics.mean_absolute_error(
-            labels, pred_values)}
+        pred_logistic = predictions['logistic']
+        pred_class = predictions['class_ids']
+        return {
+            'accuracy': tf.metrics.accuracy(labels,  pred_class),
+            'auc': tf.metrics.auc(labels, pred_logistic),
```

Comment:
Maybe call this auc_roc and make the curve explicit so as to make clear the difference with the metric below?

Reply:
Still outstanding.  If you think the code is better without, this is ok to resolve without changing anything.